runtime/src/chpl-cache.c

```diff
@@ -3744,6 +3778,8 @@ void chpl_cache_comm_prefetch(c_nodeid_t node, void* raddr,
             /* addr */ NULL, node, (raddr_t)raddr, size,
             /* sequential_readahead_length */ 0,
             CHPL_COMM_UNKNOWN_ID, ln, fn);
+
+  // TODO: record prefetches somewhere
```

Comment:
Seems vague.  If I weren't reviewing diags-related changes specifically, I might not realize that this was about diags.  Maybe "record prefetches somewhere in diags"?